example/cavity/precice-config.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
+
+<precice-configuration>
+
+    <log>
+        <sink
+          filter="%Severity% > debug and %Rank% = 0"
+          format="---[precice] %ColorizedSeverity% %Message%"
+          enabled="true" />
+    </log>
+
+    <solver-interface dimensions="3">
+
+    <data:vector name="Force"/>
+    <data:vector name="DisplacementDelta"/>
+
+    <mesh name="Fluid_Nodes">
+        <use-data name="DisplacementDelta"/>
+    </mesh>
+
+    <mesh name="Fluid_CellCenters">
+        <use-data name="Force"/>
+    </mesh>
+
+    <mesh name="Structure_Nodes">
+        <use-data name="DisplacementDelta"/>
+        <use-data name="Force"/>
+    </mesh>
+
+
+    <participant name="Fluid_Solver">
+        <use-mesh name="Fluid_Nodes" provide="yes"/>
+        <use-mesh name="Fluid_CellCenters" provide="yes"/>
+        <use-mesh name="Structure_Nodes" from="Structure_Solver"/>
+        <write-data mesh="Fluid_CellCenters" name="Force"/>
+        <read-data mesh="Fluid_Nodes" name="DisplacementDelta"/>
+        <mapping:rbf-volume-splines
+            direction="write"
+            from="Fluid_CellCenters"
+            to="Structure_Nodes"
+            constraint="conservative"
+            y-dead="true"/>
```

Comment:
Is this a fully 3D or a pseudo-2D case? I understand it is meant as a fully-3D case, in which case I am not sure I understand the need for `y-dead`.